css/style.css

```diff
@@ -16,4 +18,293 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+p {
 
+margin-top: 1 rem;
+margin-bottom:0;
+
+
+
+}
+
+ .header {
+
+height: 5rem;
+width: 100%;
+ }
+
+.footer{
+
+    padding: 0 1.5rem;
+    
+    margin: 0 10rem 0 10rem;
+    
+}
+
+
+/* Navigation */
+
+.nav{
+
+display:flex;
+
+
+
+}
+
+
+.logo{
+float:left;
+height:1.5rem;
+width:1.5rem;
+padding-left: 5rem;
+padding-top:1rem;
+
+}
+
+.nav_btn{
+padding:1rem;
+padding-right: 2rem;
+
+
+}
+
+.nav_btn:hover{
+colur:orange
+
+}
+
+
+
+.container_box{
+
+display:flex;
+justify-content:right;
+
+
+}
+
+ul{
+margin:0px;
+padding:2rem;
+list-style-type:none;
+    
+}
+
+li {
+    
+display:inline;
+padding-right: 2rem;
+
+}
+
+.gray{
+
+color:gray;
+
+
+}
+
+/* Hero Section */
+
+
+.hero{
+
+
+background-image: url('../img/first-background.jpg');
+background-color: aquamarine;
+
+width:100%;
+height:40rem;
+background-position: center;
+background-repeat: no-repeat;
+background-size: cover;
+position: relative;
+margin:0 ;
+margin-top: -2.5rem;
+
+
+
+}
+
+.hero-text{
+    text-align: center;
+    position: relative;
+    top:25%;
+    color:white;
+}
+
+.hero-text h1{
+
+font-size: 3rem;
+font-weight:200;
+
+}
+
+.hero-text h2{
+    text-align: center;
+    position: relative;
+    color:black;
+}
+
+.hero-text p{
+
+font-size: 2rem;
+margin-bottom: 4rem;
+font-weight:200;
+
+}
+
+
+
+#learn_btn{
```

Comment:
Don't use IDs in your CSS. You can use IDs later on for JavaScript, but when adding styles to HTML, target your elements using classes.